src/js/engagement_view/src/modules/GraphViz/calculations/link/linkCalcs.tsx

```diff
@@ -14,14 +14,14 @@ const findNode = (id: number, nodes: VizNode[]) => {
 export const calcLinkRisk = (link: LinkType, Graph: VizGraph) => {
     // console.log("LINK", link)
     let srcNode = 
-        // findNode(link.source, Graph.nodes) || 
-        findNode(link.source.name, Graph.nodes);
+        findNode(link.source as any, Graph.nodes) ||
```

Comment:
Do these need to be any types, or can we give them types?

Reply:
For now they need to be any's. I intend to review this ASAP.